linux_os/guide/system/accounts/accounts-banners/gui_login_banner/banner_etc_gdm_banner/oval/shared.xml

```diff
@@ -0,0 +1,43 @@
+<def-group>
+  <definition class="compliance" id="banner_etc_gdm_banner" version="2">
+    <metadata>
+      <title>System GUI Login Banner Compliance</title>
```

Comment:
Please use the [oval_metadata macro](https://complianceascode.readthedocs.io/en/latest/jinja_macros/oval.html?highlight=oval_metadata#template-oval_metadata)